INSTALL

```diff
@@ -13,7 +13,8 @@ Python XMP Toolkit
 ----------------------
 The short version of installation is::
 
-  python setup.py install
```

Comment:
Shouldn't `pip install` be the preferred entry point?

Reply:
OK